src/acme.rs

```diff
@@ -37,6 +37,11 @@ const DEFAULT_RETRY_INTERVAL: Duration = Duration::from_secs(1);
 const MAX_RETRY_INTERVAL: Duration = Duration::from_secs(8);
 static REPLAY_NONCE: http::HeaderName = http::HeaderName::from_static("replay-nonce");
 
+pub enum NewAccountOutput<'a> {
```

Comment:
Comment about the commit log
`ACME account login refactoring:` -> `ACME account login refactoring.`?